bundles/ranvier-commands/commands/config.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+module.exports = (srcPath) => {
+  const Broadcast = require(srcPath + 'Broadcast');
+  const Parser = require(srcPath + 'CommandParser').CommandParser;
+  const Data = require(srcPath + 'Data');
+
+  return {
+    usage: 'config <set/list> [setting] [value]',
+    command: (state) => (args, player) => {
+      args = args.trim();
```

Comment:
This line isn't necessary, some old commands have it because they were written before a change to make this unnecessary :P